cursive/src/backends/curses/n.rs

```diff
@@ -110,7 +111,12 @@ impl Backend {
             let path = CString::new(output_path).unwrap();
             unsafe { libc::fopen(path.as_ptr(), mode.as_ptr()) }
         };
-        ncurses::newterm(None, output, input);
+        ncurses::newterm(None, output, input).map_err(|_| io::Error::new(io::ErrorKind::Other, "could not call newterm"))?;
```

Comment:
Shouldn't we do something like, keep the original error also here?
```rust
ncurses::newterm(None, output, input)
    .map_err(|e| io::Error::new(io::ErrorKind::Other, format!("could not call newterm: {}", e)))?;
```

So, instead of this:
```rust
called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "could not call newterm" }
```

we'd get this:
```rust
called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "could not call newterm nul byte found in provided data at position: 0" }
```
(but of course it would be a different future error(none implemented yet), this is just because I modded the arg, to actually error anything)